From 99cf32dec46e0ab1e2d5a40601b3fd80e1cc8686 Mon Sep 17 00:00:00 2001
From: Steve Best <sbest@redhat.com>
Date: Wed, 4 Jun 2014 19:12:39 -0400
Subject: [powercap] Add to drivers Kconfig and Makefile

Message-id: <20140604191239.42955.75566.sendpatchset@ibm-p7r2-01.lab.bos.redhat.com>
Patchwork-id: 82562
O-Subject: [PATCH RHEL7.1 BZ1085575 7/8] PowerCap: Add to drivers Kconfig and Makefile
Bugzilla: 1085575
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

RHBZ#:
------
https://bugzilla.redhat.com/show_bug.cgi?id=1085575

Description:
------------
Added changes to Makefile and Kconfig to include in driver build.

Signed-off-by: Srinivas Pandruvada <srinivas.pandruvada@linux.intel.com>
Signed-off-by: Jacob Pan <jacob.jun.pan@linux.intel.com>
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

RHEL Version Found:
-------------------
7.0

Upstream Status:
----------------
http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=12cc4b3827f8cc5973f86330ccc9d9656a31bfa8

----------------
Steve Best

Proposed Patch:
---------------

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/Kconfig b/drivers/Kconfig
index 9953a42..85f897e 100644
--- a/drivers/Kconfig
+++ b/drivers/Kconfig
@@ -68,6 +68,8 @@ source "drivers/w1/Kconfig"
 
 source "drivers/power/Kconfig"
 
+source "drivers/powercap/Kconfig"
+
 source "drivers/hwmon/Kconfig"
 
 source "drivers/thermal/Kconfig"
diff --git a/drivers/Makefile b/drivers/Makefile
index 130abc1..f227397 100644
--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -96,6 +96,7 @@ obj-$(CONFIG_PPS)  += pps/
 obj-$(CONFIG_PTP_1588_CLOCK) += ptp/
 obj-$(CONFIG_W1)  += w1/
 obj-$(CONFIG_POWER_SUPPLY) += power/
+obj-$(CONFIG_POWERCAP)  += powercap/
 obj-$(CONFIG_HWMON)  += hwmon/
 obj-$(CONFIG_THERMAL)  += thermal/
 obj-$(CONFIG_WATCHDOG)  += watchdog/
-- 
1.7.1